### PR TITLE
i18n(fr): Add `guides/migrate-to-astro/from-xx`

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
- [x] [i18n(fr): Add guides/migrate-to-astro/from-create-react-app.mdx](https://github.com/withastro/docs/commit/a967d6016e71b02ff464c5f4928026b96e90a7e8) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-docusaurus.mdx](https://github.com/withastro/docs/commit/899dc5e598e675993bb205f602871bf90d688d68) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-eleventy.mdx](https://github.com/withastro/docs/commit/aecd7c2ebc18adcbe761ccd326976f66ffa868f0) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-gitbook.mdx](https://github.com/withastro/docs/commit/df46d36476a6390c1f642ae6a85bdec50b82d504) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-gridsome.mdx](https://github.com/withastro/docs/commit/662609fbb7cd1f2e22c688fc7daf4b9fee1a1e87) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-hugo.mdx](https://github.com/withastro/docs/commit/b1bdd7e73e787286642f362574158f1887a71056) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-jekyll.mdx](https://github.com/withastro/docs/commit/8ab0d940aeb8556f923fef4ed10c5af3bb967d2c) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-pelican.mdx](https://github.com/withastro/docs/commit/ca7bbbe9fa345fd15c9a12a9f6c7fb53d572acb0) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-sveltekit.mdx](https://github.com/withastro/docs/pull/8732/commits/201bf53d948df3d6f48c0ca4120f920fa07ca64f) 
- [x] [i18n(fr): Add guides/migrate-to-astro/from-vuepress.mdx](https://github.com/withastro/docs/pull/8732/commits/4c483fe4cd77e7cd2c0215dafa408157b2b195d3)

> [!NOTE]
> Gatsby, nuxtjs and nextjs will be not added.

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/migrate-to-astro/from-vuepress.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template starlight
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+Apportez vos fichiers de contenu Markdown existants à [créer des pages Markdown](/fr/guides/markdown-content/). Vous pouvez toujours profiter du [routage basé sur les fichiers](/fr/guides/routing/) en déplaçant ces documents de `docs` dans VuePress vers `src/pages/` dans Astro. Créez des dossiers avec des noms qui correspondent à votre projet VuePress existant, et vous devriez être en mesure de conserver vos URLs existantes.
+
+VuePress, ou tout autre thème que vous avez installé, a probablement géré une grande partie de la mise en page et des métadonnées de votre site pour vous. Vous pouvez lire l'article sur [la construction d'Astro Layouts en tant qu'enveloppes de page Markdown](/fr/basics/layouts/#mises-en-page-markdownmdx) pour voir comment gérer vous-même le templating dans Astro, y compris votre page `<head>`.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/migrate-to-astro/from-vuepress.mdx:60]:
```suggestion
VuePress, ou tout autre thème que vous avez installé, a probablement géré une grande partie de la mise en page et des métadonnées de votre site pour vous. Vous pouvez lire l'article sur [la construction de Layouts Astro en tant qu'enveloppes de page Markdown](/fr/basics/layouts/#mises-en-page-markdownmdx) pour voir comment gérer vous-même le templating dans Astro, y compris votre page `<head>`.
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Thanks @thomasbnt, this was a huge one! 👍🏽 

I added some suggestions:
* some missing translations,
* some rewording (mainly concerning the words order or missing articles),
* also you mistakenly translated a `:::caution` block.

I'm not sure about my proposal for `using Astro HTML templating components` (currently `les composants Astro HTML templating`)... but I think it should be reworded, this sounds weird.

**Comment by thomasbnt** [comment]
> Thanks @thomasbnt, this was a huge one! 👍🏽
> 
> I added some suggestions:
> 
> * some missing translations,
> * some rewording (mainly concerning the words order or missing articles),
> * also you mistakenly translated a `:::caution` block.
> 
> I'm not sure about my proposal for `using Astro HTML templating components` (currently `les composants Astro HTML templating`)... but I think it should be reworded, this sounds weird.

Thanks for catching some bad translating @ArmandPhilippot !

**Review by ArmandPhilippot — APPROVED** [review]
LGTM! 🚀  Thanks for your quick updates @thomasbnt!

**Review by yanthomasdev — APPROVED** [review]
LGTM! This is a big one, thanks y'all!

**Pull request merged**